docs/reference/search/search-your-data/behavioral-analytics/behavioral-analytics-overview.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[behavioral-analytics-overview]]
-== Behavioral Analytics
+== Search Analytics
 
 Behavioral Analytics is an analytics event collection platform introduced to Elastic in version *8.7.0*.
```

Comment:
The Enterprise Search service is no longer needed to run BA as of 8.10. Wonder how we should cover this here, maybe remove the 8.7.0 reference entirely?

Reply:
I would remove the version reference from this introduction. Elsewhere, I'd communicate the relevant historical information about this feature:

- Added in 8.7 to the Enterprise Search service
- Schema changed in 8.8 (upgrade guide available)
- Moved to Elasticsearch in 8.10 (Enterprise Search no longer required)

That could go in the prerequisites section, but I'm guessing a separate section called _Version history_ would be better.

Reply:
Yeah, I'm wondering if all this info does more harm than good for a new user?

Reply:
> I'm wondering if all this info does more harm than good for a new user?

++ I think a version history (reference) section at the end of the content is the right place for this.

Reply:
Maybe a collapsible would work well here!